### PR TITLE
admin/clean-up-nosql-schema

### PR DESCRIPTION
**Pull request by JacksonMaxfield**
General cleanup and some documentation additions to nosql schema.

We are getting _really_ close to starting to implement changes to database.

I would recommend checking out this branch and running `make docs` to see the render.

Examples for each field will be added as we write pre-populate scripts / add testing data to staging

### CODE REVIEW

File: docs/document_store_schema.md [hunk 1]
```diff
@@ -0,0 +1,384 @@
+# Document Store Schema
+
+## Event
```
Comment (isaacna) [inline docs/document_store_schema.md:3]: This is kind of a preference thing, but I think it'd be nice to italicize the name of the table? The size difference in font between markdown heading level 2 and 3 isn't that big, so I think italicizing or bolding would make the schema easier to read when scrolling down.

File: docs/document_store_schema.md [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+
+
+## Minutes Item
+A minutes item is anything found on the agenda / minutes. This can be a matter but it can be a presentation or budget
+file, etc
+
+### Schema
+```
+minutes_item_id: {
+    minutes_item_id: str
```
Comment (tohuynh) [inline docs/document_store_schema.md:220]: We can remove the unnecessary line 220?

File: docs/document_store_schema.md [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+    name: str
+    title: str
```
Comment (tohuynh) [inline docs/document_store_schema.md:222]: Does a minutes item that isn't tied to matter have a `title`? If it doesn't have `title`, can we remove line 222?
If the minutes item is tied to a matter, it's title would be found in `matter.title`.

File: docs/document_store_schema.md (hunk 1 above)
Comment (JacksonMaxfield) [reply docs/document_store_schema.md:3]: I will italicize all the H3s :+1:

File: docs/document_store_schema.md (hunk 2 above)
Comment (JacksonMaxfield) [reply docs/document_store_schema.md:220]: Nice catch.

File: docs/document_store_schema.md (hunk 3 above)
Comment (JacksonMaxfield) [reply docs/document_store_schema.md:222]: Good question.... I believe it does, in a sense that it is common right now for certain minutes items for be "named" something like 'Jun 16, 2019' but the official title is 'Approval of Minutes'.

Comment (tohuynh) [reply docs/document_store_schema.md:222]: Check out this [this 4/20 meeting](http://seattle.legistar.com/MeetingDetail.aspx?ID=775475&GUID=B1AC590C-3293-436C-8FEF-7EFABD82896C&Options=info|&Search=). I think the kind of minute items you mentioned are actually related to matters. For example, the Min 277 minutes item, I think, in our DB, its `matter.name` would `Min 277` and `matter.title` would be `April 13, 2020`. As I understand it the non-matter related minute items are the ones with uppercase titles like CALL TO ORDER, ROLL CALL, PRESENTATIONS, etc.  In our DB, I'm proposing these minutes items have a `name` only. For example, the CALL TO ORDER minutes item's `name` would be CALL TO ORDER.

File: docs/document_store_schema.md [hunk 4]
```diff
@@ -0,0 +1,384 @@
...
+            phrase: str
+        }
+    ]
+    matters: [
+        {
+            id: str
+            name: str
+            decision: str
+        }
+    ]
```
Comment (tohuynh) [inline docs/document_store_schema.md:36]: Could we add a `type` field? We need it to create the ACTION text in the event card: https://projects.invisionapp.com/share/UGW7LLV87VQ#/screens/407317059

File: docs/document_store_schema.md (hunk 3 above)
Comment (JacksonMaxfield) [reply docs/document_store_schema.md:222]: Gotcha. Agree!

File: docs/document_store_schema.md (hunk 4 above)
Comment (JacksonMaxfield) [reply docs/document_store_schema.md]: I added `type` and `title` fields to all collections that reference `matter` as just a standard.

**Pull request merged**